vizro-ai/mkdocs.yml

```diff
@@ -9,14 +9,16 @@ nav:
       - Run methods: pages/user-guides/run-vizro-ai.md
       - Create advanced charts: pages/user-guides/create-advanced-charts.md
       - Use different languages: pages/user-guides/use-different-languages.md
-  - Explanation:
+  - Find out more:
       - FAQs: pages/explanation/faq.md
       - Disclaimer: pages/explanation/disclaimer.md
       - Safeguard code execution: pages/explanation/safeguard.md
       - Safety in Vizro-AI: pages/explanation/safety-in-vizro-ai.md
-  - Contribute:
-      #      - Contributing: pages/contribute/contributing.md
       - Authors: pages/contribute/authors.md
+  #- Contribute:
+  #      - Contributing: pages/contribute/contributing.md
+  - Vizro:
```

Comment:
Are we still planning to have this eventually as tabs at the top, and not just as another card?

Reply:
When there's content, I think we can consider where it fits and I would probably have it as both. At present I've been careful to match each card link text to the text in the menu/tab at the top and would want to continue with that. So whenever there's a new card, there's a new tab.

Reply:
Ok, but I meant a tab in a different position (similar to Kedro Datasets, and the other plugins), showing that it's actually different docs!

Reply:
Ah, that's a nice idea, yes, that would be sensible! Let me drop that on the relevant ticket to remind me when we get to that.